### ISSUE TITLE
YAML Frontmatter in the gif seems to be outdated

### ISSUE THREAD

**Issue by cor** [elias-sundqvist/obsidian-annotator#42]
I've reproduced the flow from the gif exactly, but it did not trigger the "Annotate" button in the pane's menu. However, the following syntax did work:

```md
---
annotation-target: Pdfs/mypdf.pdf
---
```

Probably a good idea to either
1. fix the issue so that the syntax from the gif works again
2. change the gif so that it shows the syntax shown above

**Comment by elias-sundqvist** [comment]
Yeah, it might be a good idea to change the gif. 
The syntax in the gif is not invalid/outdated, but you need the Dataview plugin installed for it to work.

**Comment by cor** [comment]
Alright, updated docs should be sufficient in that case :-)

Thanks again for making this plugin! It's really helpful for my workflow.

**Comment by tarsisazevedo** [comment]
I get the same errors because the gif is misleading. Maybe change the gif or put some warning on the readme could avoid this error in the future.

**Comment by elias-sundqvist** [comment]
> I get the same errors because the gif is misleading. Maybe change the gif or put some warning on the readme could avoid this error in the future.

You're right. It's clear that many people have been confused by this already. I added a warning to the readme now.

**Comment by tarsisazevedo** [comment]
Thanks! Your plugin is amazing, thanks for that 😄

**Comment by markchiazh** [comment]
Hello, I downloaded Dataview, but the annotate option still isn't turning up. Thank you.

**Comment by ilvetz** [comment]
same here as for @markchiazh : cant'view annotate option when Dataview is installed (Obsidian 1.0)
thank you for this great plugin

**Comment by voltaireodactyl** [comment]
Also experiencing this issue, thanks for any help.

**Comment by Quaeeee** [comment]
Same with above! Running Obsidian 1.0

**Comment by Ibrambit** [comment]
@markchiazh @ilvetz @voltaireodactyl @Quaeeee 

i was writing (as mentioned in a lot of docs)
 > annotation-target:pdf-path

i solved it with
 > annotaion-target::pdf-path

**Comment by leandrocfg** [comment]
Hi, someone found how to fix it? Same here...
I have tried installing dataview, uninstall Annotator and install it again, but still no "Annotate" appears
